### ISSUE TITLE
Focus Socials

### ISSUE THREAD

**Issue by VirtualAngel** [VirtualAngel/Portfolio#2]
Under Contact Me, the socials need to have a focus for keyboard users. Maybe create an after psuedoelement with the name of the social media network underneath to focus and interact with since :focus doesn't work with the fontawesome icons.
